functorch/_src/aot_autograd.py

```diff
@@ -399,7 +399,7 @@ def filter_and_record_aliased_outs(outputs):
             # This will be more complicated when you have multiple _base tensors aliasing the same
             # underlying storage, when we eventually handle that.
             # We'll need to ensure that we generate the view off of the right base.
-            inp_storage_refs = {StorageWeakRef(inpt.storage()): idx for idx, inpt in enumerate(flat_f_args)}
+            inp_storage_refs = {StorageWeakRef(inpt._storage()): idx for idx, inpt in enumerate(flat_f_args)}
```

Comment:
Just double checking that you remember that `Tensor._storage()` gives an `UntypedStorage`, whereas `Tensor.storage()` gave a `TypedStorage`. But if these changes aren't causing any failures, then that's probably okay

Reply:
yes, I want untyped here